app-config.yaml

```diff
@@ -146,6 +146,28 @@ integrations:
         $env: BITBUCKET_USERNAME
       appPassword:
         $env: BITBUCKET_APP_PASSWORD
+      workspaces:
```

Comment:
You are right, it will need a redeploy of backstage and it would not scale well for orgs with ever growing project list. For some orgs though, projects don't change often and adding it to backstage is about just a simple pull request and a redeployment - provided redeployment is not a hassle. This could act as a simple default implementation.
I was thinking that, at a later stage, an api could be created to pull the information from bitbucket cloud at runtime. With that in mind, I could move this under `static` keyword to make it obvious.

Reply:
Or it could be a separate entity in backstage e.g.

```
kind: SourceControl
spec:
  type: bitbucketcloud
```
might be an overkill, if not, then perhaps at later stage?

Reply:
Could we make an attempt at fetching this from the API straight away? Having to list out these things are going to get real tedious real quick. As freben mentioned it also doesn't mesh well with how Backstage is typically set up and deployed.